### DISCUSSION TITLE
Quarg vs Pirates

### DISCUSSION THREAD

**Discussion by movingpictures** [endless-sky/endless-sky#8271] [Balance, Mechanics, and Interactions]
Why do the Quarg blow up human pirates? 
It's a serious question: isn't that needlessly wasting life and, moreover, getting involved in human affairs?

**Comment by Hecter94** [comment]
Quarg don't have any Qualms about "wasting life", they don't really interfere until there's a real threat of extinction. 
In this case, it seems as simple as just responding to pirate hostility in kind. 

Not like every individual Quarg is going to follow the same philosophy as the entire Quarg government.

**Comment by lumbar527** [comment]
Maybe the pirates attacked first? It's a good question, though.

**Reply by DarcyManoel** [reply]
That particular behaviour should be reflected in game, currently the Quarg directly attack any ships that fall under the pirate banner. The pirates should need to do the same amount of damage to a Quarg ship as the player to upset them enough for them to attack you.
